grails-app/services/org/pih/warehouse/data/ProductSupplierService.groovy

```diff
@@ -46,12 +47,15 @@ class ProductSupplierService {
     ProductSupplierDataService productSupplierGormService
 
 
-    List<ProductSupplier> getProductSuppliers(ProductSupplierListParams params) {
+    List<ProductSupplier> getProductSuppliers(ProductSupplierFilterCommand params) {
+        if (params.hasErrors()) {
+            throw new ValidationException("Invalid params", params.errors)
+        }
```

Comment:
_Thinking out loud..._
My lazy butt is wondering if we should/could/want to create `validationInterceptor` that handles checking if a command object has errors and throw an exception instead of writing this logic everywhere ourselves.

Reply:
I thought the same thing with respect to request.exception (maybe we could handle both cases in the same interceptor). However, something doesn't feel right about this approach so I'd prefer to explore it as part of a separate spike.